src/Illuminate/Auth/Notifications/ResetPassword.php

```diff
@@ -63,8 +72,8 @@ public function toMail($notifiable)
             return call_user_func(static::$toMailCallback, $notifiable, $this->token);
         }
 
-        if (static::$createUrlCallback) {
-            $url = call_user_func(static::$createUrlCallback, $notifiable, $this->token);
+        if (! empty($this->customUrl)) {
```

Comment:
The `empty` function should generally be avoided. You should probably just do a null check here.

Reply:
A solution came out of my hat. 😃